### PR TITLE
docs(operations management) content update, review

### PR DESCRIPTION
**Pull request by aibcmars**
Relates to [ICI-1230](https://bonitasoft.atlassian.net/browse/ICI-1230)

### CODE REVIEW

File: md/monitoring.md [hunk 1]
```diff
@@ -11,38 +11,50 @@ The artificial intelligence algorithm applies a prediction to each case: its lik
 Based on this information, the page shows a pie chart with the proportion of cases *Late* (already behind the deadline), *Predicted late* (likelihood is below 50%), and *On time* (likelihood is 50% or above). 
 The decision threshold that makes cases *Predicted late* or *On time* can be configured in the [Configuration Living Application](configure.md). 
 
-It also contains a tabbed list of cases that identifies each case by its ID as well as the name of the initiator. 
+It also contains a tabbed list of cases that identify each case by its ID as well as the name of the initiator.
```
Comment (csouchet) [inline md/monitoring.md]: Why not "identifies" ? Is it not the "list" who identify each case ?

File: md/monitoring.md [hunk 2]
```diff
@@ -11,38 +11,50 @@ The artificial intelligence algorithm applies a prediction to each case: its lik
...
+  - The current business data values
 
 This helps Operations Managers make the best short-term decision to put operations back on track if needed, and organize the team's activity.
 
 ## Analytics page
 
 This page takes advantage of Bonita ICI storage, and the opportunity it gives to query archived cases database to get trends on raw indicators, as well as a comparison to Key Performance Indicators.
-This view is filtered by process and period of time, and optionally by process version.
-It is made of three parts.
+This view is filtered by a process, period of time, and optionally by process version.
```
Comment (csouchet) [inline md/monitoring.md]: Why add 'a' for 'process' and not for 'period of time' and 'process version' ?

File: md/monitoring.md (hunk 1 above)
Comment (aibcmars) [reply md/monitoring.md]: Of course, you are right - it's the list. I assumed its the list but got wrong with the conjugation.

File: md/monitoring.md (hunk 2 above)
Comment (aibcmars) [reply md/monitoring.md]: There are as many responses for that question on the internet as the possibilities:)
I've checked with Mickey and we have paraphrased the sentence: 
This view is filtered by process and period of time, and can also be filtered by process version.

**Review by csouchet — APPROVED** [review]

**Pull request merged**